GVRf/Framework/src/org/gearvrf/GVRTextureCapturer.java

```diff
@@ -127,6 +127,9 @@ public void setCapture(boolean capture, float fps) {
     public void addListener(TextureCapturerListener l) {
         synchronized (mListeners) {
             mListeners.add(l);
+            if (1 == mListeners.size()) {
```

Comment:
it might be more canonical to test for empty() before adding. but testing for 1 is totally correct.